rpcs/ledgerService.go

```diff
@@ -199,6 +191,20 @@ func (ls *LedgerService) ServeHTTP(response http.ResponseWriter, request *http.R
 		}
 	}
 	defer cs.Close()
+	if conn := ls.net.GetHTTPRequestConnection(request); conn != nil {
+		maxCatchpointFileWritingDuration := 2 * time.Minute
```

Comment:
Is this giving an additional 2 minutes?

Reply:
Yes (exactly as before). The intent here is that regardless of the file size, we would always have some "connection-establishing" timing, and add from there.